### PR TITLE
[UI][Sam] Outline UI

### PR DESCRIPTION
**Pull request by franzsamilo**
**Clickup Task:**
https://app.clickup.com/t/865curhtb

**Description:**
- Changed shadows to borders (outlines) on multiple screens

**Images/Videos:**
https://github.com/ezerssss/unihub/assets/124412484/dd38909c-bb6f-4209-890c-43c23a545c5d

### CODE REVIEW

File: components/Footer/Footer.tsx [hunk 1]
```diff
@@ -35,11 +35,8 @@ function Footer() {
   }
 
   return (
-    <View className="bg-white shadow shadow-gray-500">
-      <View
-        className="flex-wrap flex-row justify-between border-t-0 border-transparent px-5 py-2 pb-4"
-        style={{ elevation: 1 }}
-      >
+    <View className="border-t border-unihub-gray-300 border-opacity-10 bg-white">
+      <View className="flex-row flex-wrap justify-between px-5 py-2 pb-4 ">
```
Comment (ezerssss) [inline components/Footer/Footer.tsx:39]: You can merge nalang this `View` sam

so basically one View nalang that have all the `classNames`

```
<View className="flex-row border-t border-unihub.....
```

File: components/Footer/TransactionsFooter.tsx [hunk 2]
```diff
@@ -42,11 +42,8 @@ function TransactionsFooter({
   }
 
   return (
-    <View className="bg-white shadow shadow-gray-500">
-      <View
-        className="flex flex-row justify-evenly border-t-0 border-transparent pb-5 pt-2"
-        style={{ elevation: 1 }}
-      >
+    <View className="border-t border-unihub-gray-300 border-opacity-10 bg-white">
+      <View className="flex flex-row justify-evenly pb-5 pt-2">
```
Comment (ezerssss) [inline components/Footer/TransactionsFooter.tsx:46]: merge

File: components/OrderConfirmationFooter/footer.tsx [hunk 3]
```diff
@@ -4,10 +4,10 @@ import UniHubIconSmall from '../icons/UniHubIconSmall';
 
 function Footer() {
   return (
-    <View className="absolute bottom-0 flex h-48 w-full rounded-t-3xl bg-white shadow shadow-black">
+    <View className="absolute bottom-0 flex h-48 w-full rounded-t-3xl bg-white border border-unihub-gray-300 border-opacity-10">
       <View
-        className="h-full rounded-t-3xl border-t-0 border-transparent"
-        style={{ elevation: 1 }}
+        className="h-full rounded-t-3xl"
+
```
Comment (ezerssss) [inline components/OrderConfirmationFooter/footer.tsx:10]: merge

File: screens/Chat/TransactionButton.tsx [hunk 4]
```diff
@@ -184,11 +184,8 @@ export default function TransactionButton(props: PropsInterface) {
   }, [user, currentStatus, handleDenyServices]);
 
   return (
-    <View className="bg-white shadow shadow-gray-100">
-      <View
-        className="flex-row justify-between px-3 py-4"
-        style={{ elevation: 1 }}
-      >
+    <View className="bg-white border border-unihub-gray-300 border-opacity-10 ">
+      <View className="flex-row justify-between px-3 py-4">
```
Comment (ezerssss) [inline screens/Chat/TransactionButton.tsx:188]: merge

File: screens/Sell/Sell.tsx [hunk 5]
```diff
@@ -352,24 +351,16 @@ export default function Sell({ navigation }: RootNavigationProps) {
...
-          width: '100%',
-        }}
-      >
-        <View
-          className="flex h-28 w-full justify-center border-t-0 border-transparent py-4"
-          style={{ elevation: 1 }}
+
+      <View className="flex h-28 w-full justify-center border border-unihub-gray-300 border-opacity-10 py-4">
+        <TouchableOpacity
+          className="right-3 h-12 w-36 items-center justify-center self-end rounded-lg bg-secondary-100"
```
Comment (ezerssss) [inline screens/Sell/Sell.tsx:357]: can you try merging this one? Just maybe remove the View and put all the classnames sa TouchableOpacity if kaya

File: screens/SpecificProduct/SpecificProduct.tsx [hunk 6]
```diff
@@ -304,8 +304,8 @@ function SpecificProduct({ route, navigation }: ProductNavigationProps) {
       </ScrollView>
       <View className="h-[108px] bg-white shadow shadow-unihub-gray-100">
```
Comment (ezerssss) [inline screens/SpecificProduct/SpecificProduct.tsx:305]: forgot to remove the shadow?

File: screens/Transactions/Listings.tsx [hunk 7]
```diff
@@ -96,11 +96,8 @@ function Listings(props: PropsInterface) {
   );
 
   return (
-    <View className="shadow-lg" key={title}>
-      <View
-        className="relative mx-4 mb-2 h-40 flex-row rounded-2xl border-0 border-transparent bg-white p-2 px-4 py-5"
-        style={{ elevation: 4 }}
-      >
+    <View key={title}>
+      <View className="relative mx-4 mb-2 h-40 flex-row rounded-2xl bg-white p-2 px-4 py-5">
```
Comment (ezerssss) [inline screens/Transactions/Listings.tsx:100]: merge siguro, just put the key nalang inside tapos remove the outside View

File: screens/Transactions/OrderRequests.tsx [hunk 8]
```diff
@@ -36,10 +36,7 @@ export default function OrderRequests({ route }: OrderRequestsNavigationProps) {
         </Text>
       </View>
       <View className="mt-5 shadow-lg" key={title}>
```
Comment (ezerssss) [inline screens/Transactions/OrderRequests.tsx:38]: forgot to remove the shadow?

File: screens/Transactions/Transactions.tsx [hunk 9]
```diff
@@ -191,10 +191,9 @@ export default function Transactions({ navigation }: RootNavigationProps) {
       }
 
       return (
-        <View className="shadow-lg" key={title}>
+        <View key={title}>
           <TouchableOpacity
-            className="relative mx-4 mb-2 h-40 flex-row rounded-2xl border-0 border-transparent bg-white p-2 px-4 py-5"
-            style={{ elevation: 4 }}
+            className="relative mx-4 mb-2 h-40 flex-row rounded-2xl border border-unihub-gray-300 border-opacity-10 bg-white p-2 px-4 py-5"
```
Comment (ezerssss) [inline screens/Transactions/Transactions.tsx:196]: merge

**Review by ezerssss — CHANGES_REQUESTED** [review]
Make sure to remove all `shadow` classnames sam

To do so you can just `ctrl` + `shift` + `f` to find all possible shadow in all of the codebase

**Review by ezerssss — APPROVED** [review]

**Review by jedmamosto — APPROVED** [review]
Thanks Sam! GJJJJ

Approved.

**Pull request merged**